apps/ledger-live-desktop/src/renderer/components/Carousel/helpers.tsx

```diff
@@ -84,6 +84,13 @@ export const useDefaultSlides = (): {
     (index: number) => {
       if (portfolioCards && portfolioCards.length > index) {
         const slide = portfolioCards[index];
+        if (slide?.id?.includes("testinDev")) {
+          dispatch(
```

Comment:
if we take the `setPortfolioCards` statement below (line 102) out of the `if (currentCard)` condition, we won't need to repeat it here.